lab_6_pipeline/pipeline.py

```diff
@@ -173,11 +275,24 @@ def __init__(self, corpus_manager: CorpusManager, analyzer: LibraryWrapper) -> N
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self._analyzer = analyzer
 
     def run(self) -> None:
         """
         Visualize the frequencies of each part of speech.
         """
+        for article in self._corpus.get_articles().values():
```

Comment:
Interesting check. Do you have this one because of tests?

Or you may get empty files after Stanza/Udpipe processing?

Reply:
Stage 5.3 in documentation requires this check